.drone.yml

```diff
@@ -32,16 +32,24 @@ steps:
     commands:
       - apk add build-base git libffi-dev cargo
       - pip3 install --upgrade setuptools
+      - cd opencti-platform
+      - OPENCTI_COMMIT=$(git rev-parse --short HEAD)
+      - echo [INFO] using opencti@$DRONE_COMMIT_BRANCH:$OPENCTI_COMMIT
```

Comment:
just an improvement on the way, to finally have a clear feedback on what is the precise version used.